.github/workflows/release_pr.yml

```
@@ -0,0 +1,50 @@
+name: Create PR for a release
```

This does more than just create the PR now - it also creates the draft release.  In the future, I think we could add to this workflow open a PR in the docs repo to increment the version there as well.

```suggestion
name: Prepare Next Release
```